dftimewolf/lib/collectors/gcloud.py

```diff
@@ -31,8 +32,15 @@ class GoogleCloudCollector(module.BaseModule):
         VM will be based.
     all_disks (bool): True if all disks attached to the source
         instance should be copied.
+    ANALYSIS_VM_CONTAINER_ATTRIBUTE_NAME (str): The name of the TicketAttribute
```

Comment:
You don't need to add these as attributes - and shouldn't these be private constants anyway, ie. _ANALYSIS_VM_CONTAINER_ATTRIBUTE_NAME ?

Reply:
They can indeed be private since the goal is to subclass this and override these attribute names.